client/my-sites/exporter/guided-transfer-card/complete-purchase-notice.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+import { cartItems } from 'lib/cart-values';
+import upgradesActions from 'lib/upgrades/actions';
+import page from 'page';
+
+class CompletePurchaseNotice extends Component {
+	redirectToCart = () => {
+		upgradesActions.addItem( cartItems.guidedTransferItem() );
+		page( `/checkout/${ this.props.siteSlug }` );
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<Notice
+				status="is-warning"
+				showDismiss={ false }
+				text={ translate(
+					`It looks like you've started a Guided Transfer.
```

Comment:
>This violates wpcalypso/i18n-no-newlines (those line breaks are respected in a template string).

I see you fixed them up for the changes implemented here, but there's a few other issues in `exporter/notices.jsx` which should be cleaned up too (here or in a separate pull request).